### PR TITLE
Add Jordan as a mentor

### PR DESCRIPTION
**Pull request by zinyando**
Add Jordan as a mentor

### CODE REVIEW

File: app/utils/mentors.js [hunk 1]
```diff
@@ -95,11 +95,18 @@ export default [{
...
     twitter: 'holtbt',
     image: '/media/mentors/brian-holt.jpg',
     frameworks: ['react'],
     bio: 'UI architect, React and Redux instructor for Frontend Masters. Netflix and Reddit alum, conference speaker and trainer. '
+  },
+  {
+    name: 'Jordan Morano',
+    twitter: 'jordan_morano',
+    image: '/media/mentors/jordan-morano.jpg',
+    frameworks: ['react', 'Ember'],
```
Comment (taras) [inline app/utils/mentors.js]: Can you please add `elixir` to this list and make all of the items lower case.

Comment (zinyando) [inline app/utils/mentors.js]: @taras what about Ruby on Rails?

Comment (taras) [inline app/utils/mentors.js]: No, it's ok. It's not shiny

**Comment by zinyando** [comment]
I've made the change you suggested

**Comment by taras** [comment]
Please deploy it

**Pull request merged**